src/otx/algorithms/common/adapters/mmcv/configurer.py

```diff
@@ -59,15 +80,46 @@ def configure(
         if cfg.model_task != self.task:
             raise ValueError(f"Given cfg ({cfg.filename}) is not supported by {self.task} recipe")
 
+        self.merge_configs(cfg, data_cfg, data_pipeline_path, hyperparams_from_otx, **kwargs)
+
         self.configure_ckpt(cfg, model_ckpt_path)
-        self.configure_data(cfg, data_cfg)
         self.configure_env(cfg)
         self.configure_data_pipeline(cfg, input_size, model_ckpt_path, **kwargs)
         self.configure_recipe(cfg, **kwargs)
         self.configure_model(cfg, data_classes, model_classes, ir_options, **kwargs)
+        self.configure_hooks(
+            cfg,
+        )
         self.configure_compat_cfg(cfg)
         return cfg
 
+    def merge_configs(self, cfg, data_cfg, data_pipeline_path, hyperparams_from_otx, **kwargs):
+        """Merge model cfg, data_pipeline cfg, data_cfg, and hyperparams from otx cli."""
+
+        logger.info("merge_configs()")
```

Comment:
```suggestion
        logger.debug("merge_configs()")
```
How about to change it as `debug`? It's just my opinion, so if you think `info` is more appropriate, just resolve it.